Update auto-generated documentation

snoopycrimecop opened the pull request:
Repository: openmicroscopy/ome-documentation
Already up-to-date.



Generated by build [OMERO-DEV-latest-docs-autogen#895](https://ci.openmicroscopy.org/job/OMERO-DEV-latest-docs-autogen/895/).

----
--no-rebase

jburel commented:
I will run again the job post release acceptance since the history PR could not be included until today due to travis failure

snoopycrimecop committed:
Update auto-generated documentation

Repository: openmicroscopy/ome-documentation
Already up-to-date.

Generated by OMERO-DEV-latest-docs-autogen#895 (https://ci.openmicroscopy.org/job/OMERO-DEV-latest-docs-autogen/895/)
